src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -117,6 +117,9 @@
  */
 public interface Message extends ISnowflake, Formattable
 {
+    /** Template for {@link #getJumpUrl()}.*/
+    String JUMP_URL = "https://discord.com/channels/%s/%s/%s";
```

Comment:
If this is to be used by developers outside of the method it was intended for them we need to document the replacement parameters

Reply:
if that were to be done, we'd have to change it for all url templates.

and also, how would you want the docs for it to look like?

Reply:
Yeah, you make a good point.
I'd be okay not including it in this PR as it is an expansion of scope.

However, something like this might be good:
```
[0] - {guildId} or "@me"
[1] - {channelId}
[2] - {messageId}
```